Sources/GDBRemote/Session.cpp

```diff
@@ -24,6 +24,8 @@
 
 #if defined(OS_LINUX)
 #define UNPACK_ID(STR) std::strtoul(STR, nullptr, 10)
+#elif defined(__FreeBSD__)
+#define UNPACK_ID(STR) std::strtoul(STR, nullptr, 10)
```

Comment:
No, I don't think there's any reason. I can't remember now why I did it that way. Anyhow, maybe what we need is OS_POSIX in places like that?